decidim-core/spec/forms/notifications_settings_form_spec.rb

```diff
@@ -188,7 +189,8 @@ module Decidim
 
       context "when vapid secrets are not present" do
         before do
-          Rails.application.secrets.delete(:vapid)
+          allow(ENV).to receive(:fetch).and_call_original
+          allow(ENV).to receive(:fetch).with("VAPID_PUBLIC_KEY", nil).and_return("")
```

Comment:
Here the same, let's do a stub for `Decidim.vapid_public_key`

Reply:
Fixed in 22115a5511